torchvision/ops/ciou_loss.py

```diff
@@ -12,6 +13,9 @@ def complete_box_iou_loss(
 ) -> torch.Tensor:
 
     """
+    # Original Implementation from
+    https://github.com/facebookresearch/detectron2/blob/main/detectron2/layers/losses.py
```

Comment:
I would leave this as a comment on the source. Long unrendered URLs are not particularly helpful for the documentation. We should make the same change on diou.

Reply:
Sphinx agrees with you :smiley: 

![image](https://user-images.githubusercontent.com/47158509/167810995-4536cbb2-f127-4086-ae86-cd1547e0a36c.png)


Let me re-wrtie

Reply:
Yeap. That looks ugly and it's on multiple places. If you want bring a separate quick PR that moves attributions on the main part of the methods to avoid the issue.

Reply:
One option is to embed the link e.g.

> Original Implementation from [Detectron2](https://github.com/facebookresearch/detectron2/blob/main/detectron2/layers/losses.py)

But the docstring should at least start by describing what the object is, even if it's very obvious from its name already. So I would suggest to write something like

```py

"""Complete Box IoU Loss.

Implementation is adapted from `Detectron2 <https://github.com/facebookresearch/detectron2/blob/main/detectron2/layers/losses.py>`__.

...
```

Reply:
![image](https://user-images.githubusercontent.com/47158509/167823496-ff12272d-2fe0-459c-b850-be464911eb2c.png)


That's how it's currently over main branch. I would suggest adding 3 words in end implementation adapted from [Detectron2](https://github.com/facebookresearch/detectron2/blob/main/detectron2/layers/losses.py).

Reply:
Will be tackling this in seperate PR anyways to unify all stuff. I feel we need bit more revamp for Ops docs.

Reply:
We should definitely provide attribution to other projects when we use code from them. Let's keep the reference on the code as a comment on the main body of the method similar to what we do in most places instead of placing a link. We can review this on the future and make changes in a coordinated manner.